src/test/java/school/hei/haapi/integration/AttendanceIT.java

```diff
@@ -0,0 +1,241 @@
+import school.hei.haapi.endpoint.rest.model.CourseSession;
+import school.hei.haapi.endpoint.rest.model.CreateAttendanceMovement;
+import school.hei.haapi.endpoint.rest.model.PlaceEnum;
+import school.hei.haapi.endpoint.rest.model.Student;
+import school.hei.haapi.endpoint.rest.model.StudentAttendance;
+import school.hei.haapi.endpoint.rest.model.StudentAttendanceMovement;
+import school.hei.haapi.endpoint.rest.security.cognito.CognitoComponent;
+import school.hei.haapi.integration.conf.AbstractContextInitializer;
+import school.hei.haapi.integration.conf.TestUtils;
+
+@SpringBootTest(webEnvironment = RANDOM_PORT)
+@Testcontainers
+@ContextConfiguration(initializers = AttendanceIT.ContextInitializer.class)
+@AutoConfigureMockMvc
+class AttendanceIT {
+  @MockBean
+  private SentryConf sentryConf;
+  @MockBean
+  private CognitoComponent cognitoComponent;
+
+  private static ApiClient anApiClient(String token) {
+    return TestUtils.anApiClient(token, ContextInitializer.SERVER_PORT);
+  }
+
+  @BeforeEach
+  void setUp() {
+    setUpCognito(cognitoComponent);
+  }
+
+  @Test
+  void manager_create_attendance_movement_ok() throws ApiException {
+    ApiClient manager1Client = anApiClient(MANAGER1_TOKEN);
+    AttendanceApi api = new AttendanceApi(manager1Client);
+
+    StudentAttendanceMovement actual = api.createAttendanceMovement(createAttendanceMovement());
+    StudentAttendanceMovement expected = new StudentAttendanceMovement()
+        .id("attendance1_id")
+        .place(PlaceEnum.ANDRAHARO)
+        .createdAt(Instant.parse("2021-11-08T07:30:00.00Z"))
+        .student(student1())
+        .attendanceMovementType(AttendanceMovementType.IN);
+
+    assertEquals(expected.getCreatedAt(), actual.getCreatedAt());
+    assertEquals(expected.getAttendanceMovementType(), actual.getAttendanceMovementType());
+    assertEquals(expected.getPlace(), actual.getPlace());
+    assertEquals(expected.getStudent(), actual.getStudent());
+  }
+
+  @Test
+  void manager_create_attendance_with_no_student_id() throws ApiException {
+    ApiClient manager1Client = anApiClient(MANAGER1_TOKEN);
+    AttendanceApi api = new AttendanceApi(manager1Client);
+
+    assertThrowsApiException("{\"type\":\"400 BAD_REQUEST\",\"message\":\"the student with #student_id_ko doesn't exist\"}",
+        () -> api.createAttendanceMovement(createAttendanceMovementKo()));
+  }
+
+  public static CourseSession courseSession1() {
+    return new CourseSession()
+        .id("course_session1_id")
+        .course(course1())
+        .begin(Instant.parse("2021-11-08T08:00:00.00Z"))
+        .end(Instant.parse("2021-11-08T12:00:00.00Z"));
+  }
+
+  public static CourseSession courseSession2() {
+    return new CourseSession()
+        .id("course_session2_id")
+        .course(course2())
+        .begin(Instant.parse("2021-08-08T15:00:00.00Z"))
+        .end(Instant.parse("2021-08-08T17:00:00.00Z"));
+  }
+
+  public static Student student1() {
```

Comment:
Are you sure that there is no student that you can immediatly use in `TestUtils`